okta_oauth2/urls.py

```diff
@@ -4,6 +4,6 @@
 
 urlpatterns = [
     path("login/", views.login, name="login"),
-    path("oauth2/callback/", views.callback, name="callback"),
+    path("oauth2/callback", views.callback, name="callback"),
```

Comment:
I think here It would be better to use regex to avoid compromising some code in use
```suggestion
    re_path("oauth2/callback/?", views.callback, name="callback"),
```